_includes/configuration/text-color.md

```diff
@@ -22,11 +22,13 @@ These settings affect the execution of the `textcolor`. The dimensions and mappi
 
 ### `color_cols`
 
-This option allows specifying the number of columns that can appear on the grid of text colors. The number of rows is calculated based on how many text colors are supplied divided by the specified number of columns.  The `textcolor_cols` and `textcolor_rows` property has been removed in {{site.productname}} 5.
+This option allows specifying the number of columns a the grid of text colors. The number of rows is calculated based on the number of text colors are supplied divided by the specified number of columns. 
```

Comment:
Somethings not quite right here... `a the grid` and `the number of text colors are supplied` :P